docs/docs/adding-analytics.md

```diff
@@ -2,7 +2,57 @@
 title: Adding analytics
 ---
 
-This is a stub. Help our community expand it.
+### Why use analytics?
 
-Please use the [Gatsby Style Guide](/docs/gatsby-style-guide/) to ensure your
-pull request gets accepted.
+Once you have your site live you will start wanting to get an idea of how many visitors are coming to your site along with other metrics such as:
+
+- What pages are most popular?
+- Where do my visitors come from?
+- When do people visit my site?
+
+Google Analytics provides a way to collect this data and perform analytics on it answering the above questions among many others. The platform is free for 10 million hits per month per Tracking ID.
+
+The Tracking ID is what is used to identify data with your site's traffic. You would typically use a different Tracking ID for each website you are monitoring.
+
+### Setting up analytics with Google Analytics
+
+First step is to set up a Google Analytics account. You can do that [here](https://analytics.google.com/) by signing in with your Google Account.
+
+Google also has a [get started page](https://support.google.com/analytics/answer/1008015?hl=en) get started page for reference.
+
+Once you have an account you will be prompted to set up a new property. This property will have a Tracking ID associated with it. In this case the property will be the website itself. Fill out the below form with your website name and URL
+
+Once complete you should now have a Tracking ID take note of it as your website will need to reference it when sending page views to Google Analytics. It should be in the format `UA-XXXXXXXXX-X`.
+
+You can find this tracking ID later by going to `Admin > Tracking Info > Tracking Code`.
+
+Now to configure Gatsby to send page views to your Google Analytics account.
+
+We are going to use `gatsby-plugin-google-analytics`
```

Comment:
Maybe a comment on what other Google plugins exist (gtag, tag manager)?

Reply:
added anchor link to other analytics options